DeveloperReadMe.txt

```diff
@@ -7,11 +7,11 @@ Source code may be found at: https://github.com/OutpostUniverse/OP2MapImager.
 
 OP2MapImager is written in C++ and the solution/project files are built for Visual Studio 2017. C++11 and C++14 features are used. The currently expiremental C++ standard library filesystem is used and is supported by both VS2017 and GCC. 
 
-OP2MapImager depends on the project OP2Utility. OP2Utility requires a Windows machine to compile due to use of windows.h. In particular, the archive read/write code depends on Windows.h. There is some desire to remove the Windows specific code, but the efforts are not yet completed. The project must be compiled for x86. 
+OP2MapImager depends on the project OP2Utility. The project must be compiled in x86 or x64. 
```

Comment:
The "must" can be updated to "may".

Reply:
Thanks for reading through the changes. I noticed another needed change while fixing this.